### PR TITLE
rm extra pgs component 

### PR DESCRIPTION
**Pull request by brittany-johnson**
https://github.com/openstax/fossil/issues/8

### CODE REVIEW

File: src/content/App.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ class App extends Component {
 
         <div className="right-panel">
           {
-            this.state.book && <Book key={this.state.book.key} data={this.state.book} />
```
Comment (brittany-johnson) [inline src/content/App.js]: I removed `this.state.book` because it doesn't seem like it was doing anything? I think?

**Review by tomjw64 — APPROVED** [review]

File: src/content/App.js (hunk 1 above)
Comment (helenemccarron) [reply src/content/App.js]: You need to define the state of the book or this happens 

^_^

File: README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `npm install`
```
Comment (helenemccarron) [inline README.md]: It should be `yarn install`, the project was set up using yarn.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
+
+Installs all necessary deppendencies listed in `package.json`.
+
 ### `npm start`
```
Comment (helenemccarron) [inline README.md]: You may want to change this to `yarn start`, which will work as well and is more consistent with the yarn setup.

**Review by helenemccarron — CHANGES_REQUESTED** [review]

**Pull request closed without merging**